### ISSUE TITLE
p-trend documentation

### ISSUE THREAD

**Issue by modedec** [isubirana/compareGroups#11]
The documentation of the "compareGroups" function regarding p-trend, is not conclusive: 

>  If the row-variable is categorical, the p-value for trend is computed as 
1-pchisq(cor(as.integer(x),as.integer(y))^2*(length(x)-1),1) 
where 'x' is the row-variable and 'y' is the grouping variable. 

In the case of categorical row variables, a chi2 test for trend is applied by the package, which is fine. However, the documentation is not clear nor do I get the same result by applying the above equation (the one in the quote).

Plase change the documentation to:  

> The p-value for trend is computed from the Pearson test when row-variable is normal, from the Spearman test when it is continuous non normal and from the chi2 test for trend when it is categorical.

**Comment by isubirana** [comment]
In fact, the p.trend for categorical variables is computed in the same way SPSS does for linear-by-linear chi-square association for crosstabs. Searching in SPSS help it is said that a Mantel-Haenszel test of trend is used (see [IPSS help page](https://www.ibm.com/support/pages/mantel-haenszel-test-trend-available-crosstabs)). So, I have changed the text in the `compareGroups` help documentation to 

```
If the row-variable is categorical, the p-value for trend is computed from Mantel-Haenszel test of trend.
``` 
By now, I have modified it in the lattest github version.
